.github/workflows/create-release.yaml

```diff
@@ -54,11 +54,6 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Create changelog
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./scripts/create_changelog.sh ${{ github.event.inputs.name }}
-
```

Comment:
I'm also not sure about a scenario where someone might mess with the labels on an already merged PR. How will changelog generation be handled during the release process?